### PR TITLE
First pass at Client CLI

### PR DESCRIPTION
**Pull request by ihodes**
Adds describe and status to the CLI

### CODE REVIEW

**Comment by ihodes** [comment]
cf https://github.com/hammerlab/coclobas/issues/35

File: src/lib/command_line.ml [hunk 1]
```diff
@@ -71,6 +92,40 @@ let root_term () =
...
+  let term =
+    let open Term in
+    pure (fun (`Base_url base_url) action ids ->
+        let res = client ~base_url action ids in
+        match Lwt_main.run res with
+        | `Error e ->
+          eprintf "An error has occured.\n";
+          exit 2
+        | `Ok () -> ()
+      )
```
Comment (smondet) [inline src/lib/command_line.ml]: There is a `run_deferred` function that display the errors.
https://github.com/hammerlab/coclobas/blob/master/src/lib/command_line.ml#L54

Comment (ihodes) [inline src/lib/command_line.ml]: Ah right I meant to improve this; unfortunately the types don't match for the run_deferred function to work here, as the errors are different.

Comment (smondet) [inline src/lib/command_line.ml]: Then you should update `Error.to_string` :)

**Comment by ihodes** [comment]
Back to you!

**Pull request merged**